docker-compose.yaml

```diff
@@ -82,6 +82,6 @@ services:
     container_name: "${PROJECT_NAME}_traefik"
     command: -c /dev/null --web --docker --logLevel=INFO
     ports:
-      - '80:80'
+      - '${TRAEFIK_NGINX_PORT:-80}:80'
```

Comment:
As this change is about Traefik, not Nginx, the variable name should be changed.